cpp/src/arrow/json/options.h

```diff
@@ -65,6 +65,8 @@ struct ARROW_EXPORT ReadOptions {
   /// Block size we request from the IO layer; also determines the size of
   /// chunks when use_threads is true
   int32_t block_size = 1 << 20;  // 1 MB
+  /// weather we will use a separate thread for read ahead
```

Comment:
Can you update the comment to match the one in the CSV options?

Reply:
Done